test/images/network-tester/webserver.go

```diff
@@ -227,32 +228,46 @@ func contactOthers(state *State) {
 		log.Printf("Server version: %#v\n", v)
 	}
 
+
+	for start := time.Now(); time.Since(start) < waitTimeout; time.Sleep(10 * time.Second) {
+		eps := getWebserverEndpoints()
```

Comment:
why not just do this in the test? I'd rather put as little as possible into a container that runs in the cluster during an e2e. Plus if you wanted to, for example, change the timeout you'd have to rebuild the image. Doing this in the container also makes surfacing errors hard, and reduces the scope of debug information you can dump in logs if something goes wrong. 

How hard is it to tell this container: hit this endpoint (by ip) and wait for the result? and repeat for all endpoints, fail if one fails and dump diagnostic info in the test